include/libopencm3/stm32/common/rtc_common_l1f024.h

```diff
@@ -426,6 +426,20 @@ specific memorymap.h header before including this header file.*/
 #define RTC_ALRMXSSR_SS_SHIFT     (0)
 #define RTC_ALARXSSR_SS_MASK      (0x7fff)
 
+enum rtc_fmt {
+	RTC_CR_FMT_24H = 0x0,
+	RTC_CR_FMT_12H,
+};
+
+enum rtc_wdu {
```

Comment:
what's wrong with rtc_weekday?

Reply:
Just naming the enum differently? I'm fine with this :+1: